addons/survey/models/survey_survey.py

```diff
@@ -727,19 +730,35 @@ def _get_session_most_voted_answers(self):
                 if not most_voted_answer or votes_by_answer[most_voted_answer] < votes_by_answer[answer]:
                     most_voted_answer_by_questions[question] = answer
 
-        # return a fake 'audiance' user_input
+        # return a fake 'audience' user_input
         fake_user_input = self.env['survey.user_input'].new({
             'survey_id': self.id,
+            'predefined_question_ids': [(6, 0, self._prepare_user_input_predefined_questions().ids)]
         })
 
-        fake_user_input_lines = self.env['survey.user_input.line']
+        # Only take questions that are triggering other questions, not the rest that is still user specific.
+        triggering_answer_by_question, triggered_questions_by_answer, selected_answers = fake_user_input._get_conditional_values()
+        triggering_questions = set(answer.question_id for answer in triggered_questions_by_answer.keys())
+
+        User_input_line = self.env['survey.user_input.line']
+        fake_user_input_lines = User_input_line
         for question, answer in most_voted_answer_by_questions.items():
-            fake_user_input_lines |= self.env['survey.user_input.line'].new({
-                'question_id': question.id,
-                'suggested_answer_id': answer.id,
-                'survey_id': self.id,
-                'user_input_id': fake_user_input.id
-            })
+            if question in triggering_questions:
+                fake_user_input_lines |= User_input_line.new({
```

Comment:
yep ! because in session you are part of an audience. and that's the audience that decides which path we will follow.
So if there are subquestion after Most anticipated feature that depends from CRM list view, it would be weird to show your answer and sub questions depending of an answer you didn't answered.. well it could be logic.. but.. meh'..
"Only take, as most voted answers, answers from question that are triggering
other questions, because only those questions will have their answers depending
on the audience. The other questions stay user specific."